static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -84,6 +96,8 @@ export const replayEventMap: Record<ReplayEventKey, string | null> = {
   'replay.details-tab-changed': 'Changed Replay Details Tab',
   'replay.details-time-spent': 'Time Spent Viewing Replay Details',
   'replay.details-viewed': 'Viewed Replay Details',
+  'replay.issues-detail-count': 'Issues Detail Replay Count',
```

Comment:
Could this be phrased like "Render ..." or something. it reads like a snapshot of a stat which i think will be hard to use when it comes time to make charts.

Reply:
done!